Automattic/jetpack
# [Plugin] Backup: Add Jetpack Header and Footer

#### Changes proposed in this Pull Request:
* Adds Temporary Jetpack Header
  * This Header is a port of the [masthead](https://github.com/Automattic/jetpack/tree/master/projects/plugins/jetpack/_inc/client/components/masthead) component in the Jetpack Plugin.  That component has more functionality, but also more dependency on the Jetpack Plugin.  We'll want to look at the `masthead` there when extracting to an RNA Header component for full functionality.
  * The styles in the `/src/masthead` folder added in this commit can be removed when we port an RNA Header component, and shouldn't be depended upon for other components.

* Adds Jetpack RNA Footer

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Install Backup Plugin
* Ensure that header and footer appear as in the screenshot
<img width="1540" alt="Screen Shot 2021-07-15 at 1 14 01 PM" src="https://user-images.githubusercontent.com/1992658/125829493-ae561834-9acb-429f-9cbd-8454f88f3280.png">
